contrib/macdeploy/detached-sig-create.sh

```diff
@@ -23,7 +23,7 @@ fi
 rm -rf ${TEMPDIR} ${TEMPLIST}
 mkdir -p ${TEMPDIR}
 
-${CODESIGN} -f --file-list ${TEMPLIST} "$@" "${BUNDLE}"
+${CODESIGN} --options runtime -f --file-list ${TEMPLIST} "$@" "${BUNDLE}"
```

Comment:
After reading through the [docs](https://developer.apple.com/documentation/xcode/notarizing_macos_software_before_distribution) it seems that passing `--timestamp` here would also be required for notarization; given that we are signing manually?

> Include a secure timestamp with your code-signing signature. (The Xcode distribution workflow includes a secure timestamp by default. For custom workflows, include the --timestamp option when running the codesign tool.)

Reply:
Good point. Added.